tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -725,9 +730,9 @@ def test_limit_with_orderby_is_overridden_by_paginator_limit(self):
         `limit` parameter
         """
         org_id = self.organization.id
-        tag1 = _indexer_record(org_id, "tag1")
-        value1 = _indexer_record(org_id, "value1")
-        value2 = _indexer_record(org_id, "value2")
+        tag1 = rh_indexer_record(org_id, "tag1")
```

Comment:
This is a transaction test, why are you using rh_indexer_record?

Reply:
@ahmedetefy as i mentioned early - this test send GET request to `/api/0/organizations/baz/metrics/data/` and this endpoint has https://github.com/getsentry/sentry/blob/3717db2a928b5c807dd6986992b3b103941a2c45/src/sentry/api/endpoints/organization_metrics.py#L128 `RELEASE_HEALTH` param in `get_series` function call. So if i chnage to:

```diff
diff --git tests/sentry/api/endpoints/test_organization_metric_data.py tests/sentry/api/endpoints/test_organization_metric_data.py
index 64a72c5639..eecab25acc 100644
--- tests/sentry/api/endpoints/test_organization_metric_data.py
+++ tests/sentry/api/endpoints/test_organization_metric_data.py
@@ -730,9 +730,9 @@ class OrganizationMetricDataTest(MetricsAPIBaseTestCase):
         `limit` parameter
         """
         org_id = self.organization.id
-        tag1 = rh_indexer_record(org_id, "tag1")
-        value1 = rh_indexer_record(org_id, "value1")
-        value2 = rh_indexer_record(org_id, "value2")
+        tag1 = perf_indexer_record(org_id, "tag1")
+        value1 = perf_indexer_record(org_id, "value1")
+        value2 = perf_indexer_record(org_id, "value2")
```

test won't work.